<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tvseg_ui::ConsoleDockWidget</class>
 <widget class="QDockWidget" name="tvseg_ui::ConsoleDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Console</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="tvseg_ui::ConsoleTextEdit" name="textEdit">
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tvseg_ui::ConsoleTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>consoletextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
